packages/backend-core/src/objectStore/objectStore.ts

```diff
@@ -19,7 +19,7 @@ const streamPipeline = promisify(stream.pipeline)
 const STATE = {
   bucketCreationPromises: {},
 }
-const signedFilePrefix = "/files/signed"
+export const signedFilePrefix = "/files/signed"
```

Comment:
If exporting a const like this, preference is for all upper casing name, like `SIGNED_FILE_PREFIX` - to denote a constant value rather than say a function.